src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1232,14 +1232,14 @@ public function getMimeType($format)
     /**
      * Gets the format associated with the mime type.
      *
-     * @param string $mimeType The associated mime type
+     * @param string $extendedMimeType The associated mime type
      *
      * @return string|null The format (null if not found)
      */
-    public function getFormat($mimeType)
+    public function getFormat($extendedMimeType)
```

Comment:
From the outside of this method, we always pass a MIME type, so I think that the argument should be called `$mimeType`. Internally you also check the MIME type without arguments, so maybe that could be called `$canonicalMimeType` ?